hailjwt/test/test_jwt.py

```diff
@@ -0,0 +1,12 @@
+import uuid
+import hailjwt as hj
+
+
+def test_round_trip():
+    c = hj.JWTClient(uuid.uuid4().hex)
```

Comment:
Set a good example: use Python `secrets` to generate cryptographically strong random keys: https://docs.python.org/3.6/library/secrets.html.